### PR TITLE
Replace waitress

### PR DESCRIPTION
**Pull request by Robert-Krajcik**
This replaces waitress with the familiar `shiny::withProgress()`
This addresses one small part if issue #176 Potential Features.
I have adjusted the size and position of the message box, below.

image

### CODE REVIEW

File: Server/uploadpackage.R [hunk 1]
```diff
@@ -92,29 +83,28 @@ observe({
...
       # Get and upload maintenance metrics to db.
       insert_maintenance_metrics_to_db(pkg)
-      waitress$inc(1)
       # Get and upload community metrics to db.
       insert_community_metrics_to_db(pkg)
-      waitress$inc(1)
     }
   }
+    setProgress(value = 1, detail = "-- **Uploading Packages Completed**")
+    Sys.sleep(0.25)
```
Comment (AARON-CLARK) [inline Server/uploadpackage.R:106]: Hi @Robert-Krajcik, is this `Sys.sleep(.25)` necessary? What is it doing?

File: Server/uploadpackage.R [hunk 2]
```diff
@@ -20,11 +20,10 @@ observeEvent(
...
 )
 
-#' Save all the uploaded packages, marking them as 'new', 'not found', or
-#' 'duplicate'.
-observe({
-  req(input$uploaded_file)
-  
+#' Save all the uploaded packages, 
+#' marking them as 'new', 'not found', or 'duplicate'.
+observeEvent(input$uploaded_file, {
```
Comment (AARON-CLARK) [inline Server/uploadpackage.R:25]: Hi @Robert-Krajcik. Why won't an `observe()` work here? Just curious. It looks like the only inputs are input$uploaded_file, right?

File: Server/uploadpackage.R (hunk 1 above)
Comment (Robert-Krajcik) [reply Server/uploadpackage.R:106]: gives us 1/4 second to see the "uploading packages completed" message

**Comment by Robert-Krajcik** [comment]
I am happy to have discovered a way to increase the size and font size of the progress bar.

**Comment by AARON-CLARK** [comment]
@Robert-Krajcik, thanks for the PR! When I upload my file, the browse button says, "Upload Complete" almost immediately, but it takes about about 4 seconds before the progress bar shows up. Is there anyway we can start the progress bar sooner?

image

**Comment by AARON-CLARK** [comment]
> I am happy to have discovered a way to increase the size and font size of the progress bar.

It looks very prominent!

**Comment by Robert-Krajcik** [comment]
Ready for another review @AARON-CLARK

**Review by AARON-CLARK — COMMENTED** [review]
LGTM @Robert-Krajcik.

I'll make another open issue for this, but just a heads up- when I first uploaded a csv file, I got the following prompt that stayed on the screen for about ~60 secs:

image

Afterwards, it finally moved on to uploading my packages to the db, and displayed this message in the console (I blurred out the URL but it's an internal RSPM repo): 

image


I'm not sure, but I'm guessing it has to do with the repos I have "installed":

**Pull request merged**